massa-signature/src/signature_impl.rs

```diff
@@ -25,16 +25,8 @@ use transition::Versioned;
 #[allow(missing_docs)]
 /// versioned KeyPair used for signature and decryption
 #[transition::versioned(versions("0", "1"))]
```

Comment:
We should drop the fake versions we created for testing

Reply:
Removed :)